internal/admin/dashboard/static/css/dashboard.css

```
@@ -963,6 +963,426 @@ td.col-price {
+    line-height: 1.45;
+    padding: 10px;
+    max-height: 220px;
+    overflow: auto;
+    white-space: pre;
+    color: var(--text);
+}
+
+.audit-json-body {
+    white-space: pre-wrap;
+    overflow-wrap: anywhere;
+    word-break: break-word;
```

Stylelint (17.3.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)